app/errors.py

```diff
@@ -0,0 +1,17 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    def __init__(self, message: str = "NotVaccinatedError") -> None:
+        super().__init__(message)
+
+
+class OutdatedVaccineError(VaccineError):
+    def __init__(self, message: str = "OutdatedVaccineError") -> None:
+        super().__init__(message)
+
+
+class NotWearingMaskError(Exception):
+    def __init__(self, message: str = "NotWearingMaskError") -> None:
```

Comment:
The default message for `NotWearingMaskError` could be more descriptive. Consider using a message that provides more context, such as 'Visitor is not wearing a mask.'